# Multicellular Transcriptional Landscape of Heart Failure
An integrative meta-analysis of the cardiac transcriptomie changes in heart failure. 

### Overview
We analyzed cardiac transcriptome changes captured in bulk and single-nucleus data from heart failure patients and performed an integrative analysis to identify shared multicellular programs that characterize the failing heart. In particular, we decomposed disease signatures to study the contribution of cell type composition changes.

<img src="https://www.dropbox.com/scl/fi/u8qafa0g3ohjax7ws93sp/readme_fig.png?rlkey=ypr7tnz425kc34dz3tgubsdny&raw=1" align="center" width="800">

### Key Features
1. **HF patient map**. Meta-analysis of single-nucleus data to identify shared multicellular programs across patients.
2. **Blueprint of HF**. A network of cell-type dependencies underlying the multicellular programs driving heart failure.
3. **Fibroblast atlas**. 
   Integration of fibroblast data at the single-cell level to identify distinct cell states and study their division of labor in expressing the multicellular program.
5. **Bulk integration**. Mapping multicellular programs to bulk datasets and using insights from cell type marker deregulation to deconvolute cell type proportions.
6. **Study projection**. Projecting independent datasets into the identified multicellular programs of heart failure to dissect shared deregulation events across studies.


### How to Explore the Data
We provide a user friendly access to a gene query of the the multicellular program 1 associated with heart failure in [shiny.io/](https://saezlab.shinyapps.io/reheat2/). All processed data and models can be accessed in zenodo (DOI:10.5281/zenodo.13946108)

### How to cite us
Read more at Lanzer, Ramirez et. al, BioRxiv, 2024 ([LINK](https://doi.org/10.1101/2024.11.04.621815 )). 
