# Multicellular Transcriptional Landscape of Heart Failure
An integrative meta-analysis of the cardiac transcriptomie changes in heart failure. 

### Overview
We analyzed cardiac transcriptome changes captured in bulk and single nuc data from heart failure patients and performed an integrative analysis to identify shared multicellular programs that characterize the failing heart. 
In particular we decomposed disease signatures to study the contribtution of cell type composition changes. 


### Key Features
1. **HF patient map**. Meta-analysis of single nuc data to identify shared multicellular programs
2. **Blueprint of HF**. The network of cell-type dependencies behind the multicellular program.
3. **Fibroblast atlas**. Integrating fibroblasts on single cell level to identify cell states and study their division of labor to express the multi cellular program. 
4. **Bulk integration**. We map multicellular programs to bulk and use insights form cell type marker deregulation to deconvolute cell type proportions. 
5. **Study projection**. Using independent data sets to project them into the multicellular programs of heart failure and dissect shared deregulation events. 


### How to Explore the Data
We provide a user friendly access to a gene query of the the multicellular program 1 associated with heart failure in [shiny.io/](https://saezlab.shinyapps.io/reheat2/). All processed data and models can be accessed in zenodo (DOI:)

### How to cite us
Read more at Lanzer, Ramirez et. al, biorxiv, 2024 (doi). 


This project is licensed under the [MIT License](link to license).
